easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0200-GCCcore-8.2.0-Perl-5.28.1.eb

```diff
@@ -13,6 +13,7 @@ builddependencies = [('binutils', '2.31.1')]
 dependencies = [
     ('Perl', '5.28.1'),
     ('libxml2', '2.9.8'),
+    ('pkgconf', '1.6.3'),
```

Comment:
Based on the name, I'm a bit surprised to see a pkgconf/LibPkgConf as a runtime dependency. I would have expected PkgConig::LibPkgConf to be it's own module, and this package use it as a build dep. Am I wrong?

Reply:
@vanzod Any thoughts on @Micket's question?

Reply:
@Micket No, that module only provides the binding to `libpkgconf`, that is why it is required as a dependency. By default it looks only in `/usr/local` for it, that is why the `libpkgconf` path is passed explicitly through `configopts`.

Reply:
@vanzod yes, LibPkgConf is a Perl binding for libpkgconf, but what I'm asking is couldn't it be a separate package: `PkgConfig-LibPkgConf-0.10-GCCcore-8.2.0-Perl-5.28.1.eb` and then this package could use it as a build dep?

```
# In XML-libXML config:
build_dependencies = [
   ('PkgConfig-LibPkgConf', '0.10'),
    ...
]
```
It just seemed like a package that might be useful elsewhere, and that it should only need to be a build dep for LibXML itself.

I'm just going by the name of the package though, so I might be wrong, which, in that case, feel free to dismiss this.

Reply:
@Micket At that point it would make sense to add it to the `Perl` easyconfig. What do you think?

Reply:
@vanzod But you couldn't add it as a build dep then (which is my main concern actually), so, that actually seems worse I.M.H.O.